models/curvefi/avalanche_c/curvefi_avalanche_c_trades.sql

```diff
@@ -0,0 +1,1194 @@
+{% set usdc_avalanche_c_token = "0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e" %}
+{% set usdt_avalanche_c_token = "0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7" %}
+{% set mim_avalanche_c_token = "0x130966628846bfd36ff31a822705796e8cb8c18d" %}
+{% set frax_avalanche_c_token = "0xd24c2ad096400b6fbcd2ad8b24e7acbc21a1da64" %}
+{% set renBTC_avalanche_c_token = "0xdbf31df14b66535af65aac99c32e9ea844e14501" %}
+{% set mai_avalanche_c_token = "0x5c49b268c9841aff1cc3b0a418ff5c3442ee3f3b" %}
+{% set fusd_avalanche_c_token = "0xc6636e205460be6073b65094422910161d767ef2" %}
+{% set usds_avalanche_c_token = "0xab05b04743e0aeaf9d2ca81e5d3b8385e4bf961e" %}
+{% set usdl_avalanche_c_token = "0x40b393cecf8f7d7fc79b83e8fa40e850511817f6" %}
+{% set yusd_avalanche_c_token = "0x111111111111ed1d73f860f57b2798b683f2d325" %}
+{% set nxusd_avalanche_c_token = "0xf14f4ce569cb3679e99d5059909e23b07bd2f387" %}
+{% set moremoney_avalanche_c_token = "0x0f577433bf59560ef2a79c124e9ff99fca258948" %}
+{% set agEUR_avalanche_c_token = "0x6fefd97f328342a8a840546a55fdcfee7542f9a8" %}
+{% set jEUR_avalanche_c_token = "0x9fb1d52596c44603198fb0aee434fac3a679f702" %}
+{% set eEUR_avalanche_c_token = "0xe1d70994be12b73e76889412b284a8f19b0de56d" %}
+{% set arUSD_avalanche_c_token = "0x025ab35ff6abcca56d57475249baaeae08419039" %}
+{% set axlUSDC_avalanche_c_token = "0xfab550568c688d5d8a52c7d794cb93edc26ec0ec" %}
+{% set dd_avalanche_c_token = "0x210c2e177b34145c64c4eaccd17aca07d2b86164" %}
+{% set debridge_usdc_avalanche_c_token = "0x28690ec942671ac8d9bc442b667ec338ede6dfd3" %}
+{% set defrost_h20_avalanche_c_token = "0x026187bdbc6b751003517bcb30ac7817d5b766f8" %}
+{% set ankr_aAVAXb_avalanche_c_token = "0x6c6f910a79639dcc94b4feef59ff507c2e843929" %}
+
+
+{% set usdt_e_avalanche_c_token = "0xc7198437980c041c805a1edcba50c1ce5db95118" %}
+{% set usdc_e_avalanche_c_token = "0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664" %}
+{% set dai_e_avalanche_c_token = "0xd586e7f844cea2f87f50152665bcbc2c279d8d70" %}
+{% set wbtc_e_avalanche_c_token = "0x50b7545627a5162f82a992c33b87adc75187b218" %}
+
+{% set aave_dai_avalanche_c_token = "0x47afa96cdc9fab46904a55a6ad4bf6660b53c38a" %}
+{% set aave_usdc_avalanche_c_token = "0x46a51127c3ce23fb7ab1de06226147f446e4a857" %}
+{% set aave_usdt_avalanche_c_token = "0x532e6537fea298397212f09a61e03311686f548e" %}
+{% set av3CRV_avalanche_c_token = "0x1337bedc9d22ecbe766df105c9623922a27963ec" %}
+{% set av3CRV_guage_avalanche_c_token = "0x5b5cfe992adac0c9d48e05854b2d91c73a003858" %}
+{% set aave_wbtc_avalanche_c_token = "0x686bef2417b6dc32c50a3cbfbcc3bb60e1e9a15d" %}
+{% set aave_weth_avalanche_c_token = "0x53f7c5869a859f0aec3d334ee8b4cf01e3492f21" %}
+{% set blizz_dai_avalanche_c_token = "0x6807ed4369d9399847f306d7d835538915fa749d" %}
+{% set blizz_usdc_avalanche_c_token = "0xc25ff1af397b76252d6975b4d7649b35c0e60f69" %}
+{% set blizz_usdt_avalanche_c_token = "0x18cb11c9f2b6f45a7ac0a95efd322ed4cf9eeebf" %}
+
+
+
+
+
+
+WITH dexs AS (
```

Comment:
I wonder if these individualised, per pool mappings could be made cleaner by
- using a hardcoded data table to join to
- decoding / using pool creation events to generate said table

Reply:
Thank you very much ser!

hmmm, yeah, I think that would make it cleaner and I thought about it too. The reason I opted for this approach was because it was listed as the approach (the ethereum one) we could follow in the dework bounty description. 

Although I'm not very familiar with Curve pools creation contracts, I'd not mind making an update sometime in the future and create a pool data table. I'm thinking we can use this model for other curve chains as well (Arbitrum, Polygon etc) since those would  eventually get added to dex.trades.